src/rpminspect.c

```diff
@@ -606,6 +605,13 @@ int main(int argc, char **argv)
 
     free(profile);
 
+    /* Product release specified on the command line overrides config file */
+    if (release) {
+        free(ri->product_release);
+        ri->product_release = release;
+        free(release);
```

Comment:
freeing release here will cause a dangling pointer, improper usage

Reply:
Fixed in followup commit, thanks!